Date
Thu Mar 21 11:26:01 AM EDT 2024

EnvironmentLocationNotFound: Not a conda environment: /lustre/home/debnathk/dleps/.venv/bin/python3

Traceback (most recent call last):
  File "/lustre/home/debnathk/dleps/code/DLEPS/reference_drug/RefDNN/1_nested_cv_RefDNN.py", line 6, in <module>
    import skopt
ModuleNotFoundError: No module named 'skopt'
Writing logs to the refDNN_results/ folder...Done!
Elapsed time: 1 seconds
